Form-Controls/style.css

```diff
@@ -0,0 +1,31 @@
+
+form {
+    max-width: 650px;
+    margin: 0 auto;
+    padding: 20px;
+    background-color: #1a1a2e;
+    color: #e0e0e0 ;
```

Comment:
Minor suggestion: if you find yourself repeatedly using the same color, think of creating a css variable for it.